src/WorkBC.Data/SsotContext.cs

```diff
@@ -0,0 +1,208 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using WorkBC.Data.Model.Enterprise;
+
+namespace WorkBC.Data
+{
+    public partial class SsotContext : DbContext
+    {
+        private string ssotConnectionString;
+        
+        public SsotContext(DbContextOptions<SsotContext> options, IConfiguration configuration) 
+            : base(options)
+        {
+            ssotConnectionString = configuration.GetConnectionString("SsotConnection");
+        }
+
+        public virtual DbSet<CareerProfile> CareerProfiles { get; set; }
+        public virtual DbSet<IndustryProfile> IndustryProfiles { get; set; }
+        public virtual DbSet<Noc> Nocs { get; set; }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            optionsBuilder.UseNpgsql(ssotConnectionString);
+        }
+
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            modelBuilder.Entity<CareerProfile>(entity =>
+            {
+                entity.HasKey(e => e.CareerProfileId);
+
+                entity.ToTable("edm_careerprofile");
+
+                entity.HasIndex(e => e.NocId).IsUnique();
+                
+                entity.Property(e => e.CareerProfileId).HasColumnName("careerprofileid");
+
+                entity.Property(e => e.AnnualSalarySource).HasMaxLength(255);
```

Comment:
Similarly, each column name will need to have a `.HasColumnName()` added to convert the TitleCase to lowercase.